lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -40,14 +42,26 @@ def va900_warning
       def va900_hint
```

Comment:
It's only tangentially related to this PR, but while we're in here and changing messages slightly, can you fix `#va900_warning` - specifically the nesting order of the outermost closing paren and curly brace are switched. It drives me nuts when I'm trying to parse a Sentry event that contains json within the original_body and I can't figure out what's contained in what.